src/modules/imageresizer/tests/Properties/SettingsTests.cs

```diff
@@ -271,6 +273,34 @@ public void ReloadRaisesPropertyChanged()
+
+            // Execute readFile/writefile twice and see if serialized string is still correct
+            var resultWrapper = JsonSerializer.Deserialize<SettingsWrapper>(defaultInput);
+            var serializedInput = JsonSerializer.Serialize(resultWrapper, new JsonSerializerOptions() { WriteIndented = true });
+            var resultWrapper2 = JsonSerializer.Deserialize<SettingsWrapper>(serializedInput);
+            var serializedInput2 = JsonSerializer.Serialize(resultWrapper2, new JsonSerializerOptions() { WriteIndented = true });
+
+            Assert.Equal(serializedInput, serializedInput2);
+            Assert.Equal("ImageResizer", resultWrapper2.Name);
+            Assert.Equal("1", resultWrapper2.Version);
+            Assert.NotNull(resultWrapper2.Properties);
+            Assert.True(resultWrapper2.Properties.ShrinkOnly);
+            Assert.True(resultWrapper2.Properties.Replace);
+            Assert.Equal(91, resultWrapper2.Properties.JpegQualityLevel);
+            Assert.Equal(1, (int)resultWrapper2.Properties.PngInterlaceOption);
+            Assert.Equal(1, (int)resultWrapper2.Properties.TiffCompressOption);
+            Assert.Equal("%1 %1 (%2)", resultWrapper2.Properties.FileName);
+            Assert.Equal(2, resultWrapper2.Properties.Sizes.Count);
+            Assert.False(resultWrapper2.Properties.KeepDateModified);
+            Assert.Equal("Small-NotDefault", resultWrapper2.Properties.Sizes[0].Name);
+        }
```

Comment:
I'm not sure this test is appropriate here. First of all, we test an external function. I understand that it tests if attributes are appropriately assigned and `WrappedJsonConverter` is implemented correctly.

Reply:
Also, why don't we assert strings here?

Reply:
This should test if the previous version of the settings file could be correctly read.

Reply:
Instead of asserting each property individually, we can replace all asserts by `Assert.Equal(defaultInput, serializedInput2);`. I think we need to extend default input a little bit though. Does it make sense?

Reply:
I really don't want to change defaultInput, since that is the input of the current settings file and this serialization is having a different order (Which functionally should not matter).

We can add one with a compare to a validated string though!